src/Animal.java

```diff
@@ -13,10 +13,10 @@ public Animal(int maxRunDistance, int maxSwimDistance) {
     }
```

Comment:
Class' members should have a specific order:

constants
static variables
instance variables
constructors
getters + setters
public methods
private methods
Please change this class' members order according to the info in this comment.